Add section with TLS/SSL detail

brnkrygs opened the pull request:
Add a section under "Using the MQTT protocol directly" describing how to configure a client for TLS/SSL, including links to where to get the necessary certificates, as well as example (python) code for establishing MQTT connections.

brnkrygs committed:
Add section with TLS/SSL detail

Add a section under "Using the MQTT protocol directly" describing how to configure a client for TLS/SSL, including links to where to get the necessary certificates, as well as example (python) code for establishing MQTT connections.

zolvarga commented on articles/iot-hub/iot-hub-mqtt-support.md:
```diff
@@ -73,6 +73,34 @@ If a device cannot use the device SDKs, it can still connect to the public devic
+  
+path_to_root_cer = "...local\\path\\to\\digicert.cer"
+device_id = "<device id from device registry>"
+sas_token = "<generated SAS token>
```
><generated SAS token> [](start = 13, length = 21)

Closing quotation mark is missing

zolvarga commented on articles/iot-hub/iot-hub-mqtt-support.md:
````diff
@@ -73,6 +73,34 @@ If a device cannot use the device SDKs, it can still connect to the public devic
+
+An example of how to implement this using the Python version of the [Paho MQTT library][lnk-paho] by the Eclipse Foundation might look like this:
+
+```
````
Add instruction how to download paho mqtt library:
"pip install paho-mqtt"

zolvarga reviewed (approved):
:shipit:

brnkrygs committed:
Apply editing suggestions from PR feedback

Add missing double-quote.
Add instructions on how to install paho-mqtt for python.
